### DISCUSSION TITLE
Connecting to AWS Document DB

### DISCUSSION THREAD

**Discussion by bakshigit** [mongodb/mongo-ruby-driver#2698] [General]
Hello,

We have been testing Mongobetween Proxy which internally uses this mongo-ruby-driver.

When using ‘mongobetween’ as a proxy with AWS Document DB we are unable to do ‘Insert/Update/delete’ collections in the database programmatically (Nodejs). Only find/query is working.

However, if we use ‘mongodb’ as our backend database everything seems to work. We have tried different combinations of golang drivers with Docdb, but no luck so far.

Has anyone used this driver with AWS Document DB ? Any setting needs to made in driver settings and/or on  the Database side ?

Appreciate any help.

**Comment by alexbevi** [comment]
Hi @bakshigit, unfortunately AWS Document DB is only very loosely compatible with MongoDB (as can be seen in https://www.isdocumentdbreallymongodb.com/). As the [official Drivers](https://www.mongodb.com/docs/drivers/) were developed to work with MongoDB there's little we can do in this situation.
